examples/tasks/advanced/images/image_LICENSE.txt

```diff
@@ -3,7 +3,7 @@ licensed accordingly to the original license:
 
 Project                 License                   File
 ----------------------------------------------------------------------------
-Enthought               BSD 3-Clause              LICENSE.txt
+Enthought               Proprietary
```

Comment:
I spoke with @eric-jones and @cgodshall and we don't need any license for our logo files. I have removed the entry altogether.